src/components/buttons.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
```

Comment:
- Hey, please, kindly remove these commented pieces of code to make your codebase clean and maintainable.